js/todoapp.js

```diff
@@ -0,0 +1,171 @@
+const demoTodo = [
+  {
+    name: "Follow Jijuzo on GitHub",
+    id: 1,
+    completed: false,
+  },
+];
+
+//**Todo creation**
+const todos = document.querySelector(".todo-ul");
+function addingTodo(todo) {
+  const newTodo = document.createElement("li");
+  newTodo.className = "mycheck";
+  newTodo.setAttribute("completed", todo.completed);
+  newTodo.innerHTML = `
+            <input id="${todo.id}" type="checkbox" class="check">
+            <label class="task-label" for="${todo.id}">${todo.name}</label>
+            <i class="fa-solid fa-trash-can completed-trash"></i>
+    `;
+  todos.appendChild(newTodo);
+}
+
+// **using demo todo**
+let tasks = JSON.parse(localStorage.getItem("tasks")) || demoTodo;
+function displayTodos() {
+  for (let task of tasks) {
+    addingTodo(task);
+  }
+}
+
+displayTodos();
+
+// **Creating and adding todo**
+const input = document.querySelector(".task-input");
+const addButton = document.querySelector(".add-button");
+addButton.addEventListener("click", () => {
+  if (input.value) {
+    const randomID = Math.random();
+    const userInput = {
+      name: input.value,
+      id: randomID,
+      completed: false,
+    };
+
+    tasks.push(userInput);
+    localStorage.setItem("tasks", JSON.stringify(tasks));
```

Comment:
This piece of logic (settings tasks in ls) is duplicated

Consider creating an abstraction